modules/engines/engine-hdb/src/main/java/com/sap/xsk/hdb/ds/transformer/hdbdd/HdbddTransformer.java

```diff
@@ -114,8 +126,52 @@ public XSKDataStructureHDBTableModel transformEntitySymbolToTableModel(EntitySym
 
     tableModel.setColumns(tableColumns);
     tableModel.setLocation(location);
-    if (entitySymbol.getAnnotation(CATALOG_ANNOTATION) != null) {
+    if (entitySymbol.getAnnotation(CATALOG_ANNOTATION) != null){
       tableModel.setTableType(entitySymbol.getAnnotation(CATALOG_ANNOTATION).getKeyValuePairs().get(CATALOG_OBJ_TABLE_TYPE).getValue());
```

Comment:
Move the arguments of `setTableType(...)` to a separate variable for better readability